nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/installation.md

```diff
@@ -47,3 +53,81 @@ To stop nextflow from looking for updates online, you can tell it to run in offl
 ```bash
 export NXF_OFFLINE='TRUE'
 ```
+<<<<<<< HEAD
```

Comment:
That should leave ;-)

Reply:
thought I fixed the merging conflict but I guess not :woman-facepalming: